crates/ruff_python_ast/src/nodes.rs

```diff
@@ -2108,6 +2106,439 @@ impl From<BytesLiteral> for Expr {
     }
 }
 
+bitflags! {
```

Comment:
Not sure if all this belongs in `nodes.rs`. `ruff_python_ast::str.rs` didn't feel right, though, since `nodes.rs` depends on `str.rs` (it would have created a circular dependency between the two files.